Sources/MapboxNavigation/PreviewViewController.swift

```diff
@@ -0,0 +1,633 @@
+                                   right: rightInset)
+        
+        let pitch: CGFloat = 0.0
+        
+        return CameraOptions(padding: padding, pitch: pitch)
+    }
+    
+    var backButton: BackButton!
+    
+    // :nodoc:
+    public var navigationView: NavigationView {
+        view as! NavigationView
+    }
+    
+    var speedLimitView: SpeedLimitView!
+    
+    var wayNameView: WayNameView!
+    
+    var finalDestinationAnnotations: [PointAnnotation]? = nil
+    
+    var pointAnnotationManager: PointAnnotationManager?
+    
+    var cameraFloatingButton: CameraFloatingButton!
+    
+    var styleManager: StyleManager!
+    
+    // :nodoc:
+    public private(set) var presentedBottomBannerViewController: UIViewController?
+    
+    var topBannerContainerViewHeightConstraint: NSLayoutConstraint!
+    
+    // :nodoc:
+    public weak var delegate: PreviewViewControllerDelegate?
+    
+    open override func loadView() {
+        let frame = parent?.view.bounds ?? UIScreen.main.bounds
+        let navigationView = NavigationView(frame: frame)
+        navigationView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        navigationView.navigationMapView.delegate = self
+        navigationView.navigationMapView.userLocationStyle = .courseView()
+        
+        navigationView.navigationMapView.mapView.mapboxMap.onNext(.styleLoaded) { [weak self] _ in
+            guard let self = self else { return }
+            self.pointAnnotationManager = self.navigationView.navigationMapView.mapView.annotations.makePointAnnotationManager()
+            
+            if let finalDestinationAnnotations = self.finalDestinationAnnotations,
+               let pointAnnotationManager = self.pointAnnotationManager {
+                pointAnnotationManager.annotations = finalDestinationAnnotations
+                
+                self.finalDestinationAnnotations = nil
+            }
+        }
+        
+        view = navigationView
+    }
+    
+    open override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        setupBackButton()
+        setupSpeedLimitView()
+        setupWayNameView()
+        setupFloatingButtons()
+        setupTopBannerContainerView()
+        setupBottomBannerContainerView()
+        
+        setupNavigationViewportDataSource()
+        setupPassiveLocationManager()
+        setupStyleManager()
+        setupOrnaments()
+        setupGestureRecognizers()
+        subscribeForNotifications()
+        
+        state = .browsing
+    }
+    
+    open override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        
+        // Style should be applied each time to prevent usage of the styles that were set via
+        // `NavigationViewController`.
+        styleManager.applyStyle()
+    }
+    
+    func setupStyleManager() {
+        styleManager = StyleManager()
+        styleManager.delegate = self
+        // styleManager.styles = [PreviewDayStyle()]
```

Comment:
The comment lines should be removed. So the styles for `PreviewViewController` are restricted to preview day and night style, and no customized would be allowed.